mlflow/utils/autologging_utils.py

```diff
@@ -9,3 +11,62 @@ def try_mlflow_log(fn, *args, **kwargs):
         fn(*args, **kwargs)
     except Exception as e:  # pylint: disable=broad-except
         warnings.warn("Logging to MLflow failed: " + str(e), stacklevel=2)
+
+
+def param_logger(fn, args=[], kwargs=None, unlogged=[]):
+    """
+    Log parameters explicitly passed to a function.
+    :param fn: function whose parameters are to be logged
+    :param args: arguments explicitly passed into fn
+    :param kwargs: kwargs explicitly passed into fn
+    :param unlogged: parameters not to be logged
+    :return: None
+    """
+    # Names of all parameters for the function
+    all_param_names = inspect.getargspec(fn)[0]
```

Comment:
We could unpack this btw as `all_param_names, _, _, all_default_values = inspect.getargspec(fn)`